lib/WorkflowMag.groovy

```diff
@@ -124,8 +127,8 @@ class WorkflowMag {
             System.exit(1)
         }
 
-        if (params.skip_busco && params.gtdb) {
-            log.warn '--skip_busco and --gtdb are specified! GTDB-tk will be omitted because GTDB-tk bin classification requires bin filtering based on BUSCO QC results to avoid GTDB-tk errors.'
+        if (params.skip_binqc && params.gtdb) {
+            log.warn '--skip_binqc and --gtdb are specified! GTDB-tk will be omitted because GTDB-tk bin classification requires bin filtering based on BUSCO QC results to avoid GTDB-tk errors.'
```

Comment:
```suggestion
            log.warn '--skip_binqc and --gtdb are specified! GTDB-tk will be omitted because GTDB-tk bin classification requires bin filtering based on BUSCO or CheckM QC results to avoid GTDB-tk errors.'
```